.style.yapf

```diff
@@ -0,0 +1,4 @@
+[style]
```

Comment:
Do we want to replace this with the output of yapf --style-help. This would give us a template of all options